releasenotes/notes/docker-exit-c5b8c5ab4885d54c.yaml

```
@@ -0,0 +1,8 @@
+---
+enhancements:
+  - |
+    The ``docker.exit`` service check has a new tag ``exit_code``.
+  - |
+    The ``143`` exit code is considered OK by default, in addition to ``0``.
+  - |
+    The Docker check supports a parameter ``ok_exit_codes`` to allow choosing exit codes that are considered OK.
```

This will render as three enhancements.  I think it really only "counts" as one enhancement :)
```suggestion
  - |
    The ``docker.exit`` service check has a new tag ``exit_code``.
    The ``143`` exit code is considered OK by default, in addition to ``0``.
    The Docker check supports a parameter ``ok_exit_codes`` to allow choosing exit codes that are considered OK.
```